components/helloleads/sources/new-lead-added/test-event.mjs

```diff
@@ -0,0 +1,32 @@
+export default {
+	"id": "9481462",
+	"first_name": "Oliver",
+	"last_name": "Ivor",
+	"company": "T J MORRIS",
+	"designation": "Head – Marketing",
+	"email": "ivor@outlook.com",
+	"mobile": "+449989452233",
+	"phone": "+2088037044",
+	"fax": null,
+	"website": "corporate.homebargains.co.uk",
+	"address_line1": "Unit 24-32, North Mall",
+	"address_line2": null,
+	"city": "Edmonton Green",
+	"state": "London",
+	"country": "UK",
+	"postal_code": "N90EZ",
+	"notes": null,
+	"interests": "Sample - HelloLeads LMS",
+	"category": "Sample - Individual",
+	"tags": "",
+	"lead_stage": "Customer",
+	"potential": "High",
+	"deal_size": "2500000",
+	"deal_currency": "CAD",
+	"date_of_birth": null,
+	"special_event_date": null,
+	"list_name": " Sample Leads",
+	"created": "2023-10-24 22:45:56",
```

Comment:
We can see from the test event that `created` is in ISO format instead of a timestamp, so we need to use `Date.parse(item.created)` when setting `ts` in `generateMeta`.